docs/references/asset-canister.mdx

```diff
@@ -39,17 +40,136 @@ For example:
 }
 ```
 
+By default, all projects created with `dfx new` have the option to include an asset canister in the project. If an asset canister is created through `dfx`, the default configuration includes a Rust canister that uses one of the following frontend frameworks: SvelteKit, React, Vue or Vanilla JS. The user can choose the framework through the `dfx new` interactive prompt.
+
+For asset canisters created by `dfx new`, the default configuration will include:
+
+- `"entrypoint": "assets/src/index.html"`: The webpage that should be served by default. If an asset cannot be found, this `index.html` page is served as the default fallback.
+
+- `"source": [ "assets/assets", "assets/src"]`: The default subdirectories containing asset files. For assets to be served in the asset canister, their directories must be included in this `source` list.
+
+- `"type": "assets"`: Defines the canister type as `assets`.
+
 :::caution
 The asset canister can host roughly 1GB in static files. It is recommended that you distribute your files
 across multiple canisters if the total size of all your assets begins to exceed this amount. Once you exceed this
-figure, your canister may fail to upgrade.  
+figure, your canister may fail to upgrade.
 :::
 
-<br/>
+## Rust crate
+
+The [`ic-asset`](https://crates.io/crates/ic-asset) Rust crate can be used to develop your project's asset canister.
+
+Additionally, the [ic-http-certification](https://crates.io/crates/ic-http-certification) crate can be used to:
+
+- Serve certified assets from the same canister as their primary “backend” canister.
+
+- Embed assets directly into a canister’s Wasm rather than uploading them at runtime.
+
+- Create custom routing logic, such as:
+    - Serving 404 pages in multi-page apps.
+    - Serving multiple frontends from the same canister.
+    - Certify more complex caching or streaming scenarios.
+
+## Third-party options for serving assets
+
+Other third-party options for serving assets include:
+
+- [`ic-pluto`](https://crates.io/crates/ic-pluto)
+
+- [mops server](https://mops.one/server)
```

Comment:
I would call this `Motoko server` instead of `mops server` -- `mops` is just the package manager